test/unix/test_lwt_bytes.ml

```diff
@@ -496,4 +497,209 @@ let suite = suite "lwt_bytes" [
       let check = "abcaaa" = Lwt_bytes.to_string buf in
       Lwt.return check
     end;
+
+    test "bytes read" begin fun () ->
+      let test_file = "bytes_io_data" in
+      Lwt_unix.openfile test_file [O_RDONLY] 0
+      >>= fun fd ->
+      let buf = Lwt_bytes.create 6 in
```

Comment:
I recommend zeroing all new `Lwt_bytes.t`s in the test suite with `Lwt_bytes.fill`, because the contents can be anything when each buffer is first allocated.